src/app/app.module.ts

```diff
@@ -18,6 +18,8 @@ import { ProfileComponent } from './profile/profile.component';
 
 // Services
 import { AccountService } from './shared/services/account.service';
+import { DataService } from './shared/services/data.service';
+import { DummyDataService } from './shared/services/dummy-data.service';
```

Comment:
I know these are just dummy data, but I think these two `import` statements should be moved to `profile.component.ts` since it's only being used by that component right now.

So in `profile.component.ts`, you could add a `provider` in `@Component`:
```
// profile.component.ts

import { Component, OnInit} from '@angular/core';
import { DataService } from './shared/services/data.service';
import { DummyDataService } from './shared/services/dummy-data.service';
import { Item } from '../shared/model/item.model';

@Component({
  selector: 'app-profile',
  templateUrl: './profile.component.html',
  styleUrls: ['./profile.component.css'],
  providers: [DataService, DummyDataService]
})
export class ProfileComponent implements OnInit {
  ...
}
```